lib/querly/pp/cli.rb

```diff
@@ -70,6 +70,20 @@ def run_haml
           stdout.print compiler.precompiled
         end
       end
+
+      def run_erb
+        require 'better_html'
+        require 'better_html/parser'
+        load_libs
+        source = stdin.read
+        source_buffer = Parser::Source::Buffer.new('(erb)')
+        source_buffer.source = source
+        parser = BetterHtml::Parser.new(source_buffer, template_language: :html)
+        parser.ast.descendants(:erb).each do |erb_node|
+          _, _, code_node, = *erb_node
+          stdout.puts code_node.loc.source
+        end
```

Comment:
I referenced code of erb-lint to implement this.
https://github.com/Shopify/erb-lint/blob/04dad0e1a3bac5c68157810e95329929322fc349/lib/erb_lint/processed_source.rb
https://github.com/Shopify/erb-lint/blob/04dad0e1a3bac5c68157810e95329929322fc349/lib/erb_lint/linters/rubocop.rb